src/sonic-config-engine/sonic-cfggen

```diff
@@ -281,6 +289,7 @@ def main():
         redis_bcc = RedisBytecodeCache(SonicV2Connector(host='127.0.0.1'))
         env = jinja2.Environment(loader=loader, trim_blocks=True, bytecode_cache=redis_bcc)
         env.filters['sort_by_port_index'] = sort_by_port_index
+        env.filters['is_intf_exists'] = is_intf_exists
```

Comment:
config engine should not take current system state, it only parses the config db or minigraph. otherwise, it is making sonic-cfggen complicated.

Reply:
Agreed and I will remove the newly added filter. It is not required as well.